### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Question
+about: 'Please use Stack Overflow for questions #office-ui-fabric-react'
+
+---
+
+Stuff about what happens to questions here
```
Comment (JasonGore) [inline .github/ISSUE_TEMPLATE/question.md]: Did you mean to commit this?

Comment (micahgodbolt) [reply .github/ISSUE_TEMPLATE/question.md]: I just created a PR with all the files so we could edit them....wasn't ready for merge...sorry :D

**Comment by micahgodbolt** [comment]
opening up this PR so we have some files to hack on before merging

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (JasonGore) [reply .github/ISSUE_TEMPLATE/question.md]: Gotta be careful with easy looking PRs, people will want to get them in and out of the queue ;)

Comment (micahgodbolt) [reply .github/ISSUE_TEMPLATE/question.md]: hah..yup! should have tossed the label on earlier

**Comment by micahgodbolt** [comment]
@michaelangotti seems we were both working on it :) Care to take 1 more pass?

**Comment by michaelangotti** [comment]
@micahgodbolt Looks good to me. Your Question template was almost verbatim of what I had. I deleted my Feature req but it still shows it has a conflict. Other than that, we are good to go.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,19 +1,15 @@
----
+--
 name: Feature request
-about:  Do you have an idea or improvement for Fabric? Let us know.
+about: Do you have Feature or improvemnet for Fabric? Let us know.
```
Comment (JasonGore) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Typo

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -1,19 +1,15 @@
...
+**What component or utility would this be added to**
 
-**Describe alternatives you've considered:**
-<!--- Fill in or delete --->
+**Have you discussed this feature with our team, and if so, who**
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
-<!--- Fill in or delete --->
+**Additional context/screenshots**
```
Comment (JasonGore) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Could we hope to ask for even a prototype via CodePen or something similar?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -1,19 +1,15 @@
----
+--
```
Comment (JasonGore) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I think this broke the formatted output

Comment (micahgodbolt) [reply .github/ISSUE_TEMPLATE/feature_request.md]: copy paste FTL

File: .github/ISSUE_TEMPLATE/question.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+name: Question
+about: 'Please use Stack Overflow for questions'
+
+---
+
+For _how-to_ questions and other non-bugs, please use StackOverflow instead of Github issues. You can tag your questions with [office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).
+
+If you are a Microsoft employee you can also join the Micorosft UI Fabric teams channel.
```
Comment (cliffkoh) [inline .github/ISSUE_TEMPLATE/question.md]: typo on this line

**Pull request merged**